### PR TITLE
VR-3404: repo.new_commit(parents=[commit1, commit2])

### PR DESCRIPTION
**Pull request by convoliution**
~Blocked on testing until backend has `createCommit` implemented.~

### CODE REVIEW

File: client/verta/verta/_repository/repository.py [hunk 1]
```diff
@@ -90,6 +90,16 @@ def _get(cls, conn, name=None, workspace=None, id_=None):
     def init(self):
         return commit.Commit(self._conn, self.id)
 
+    def new_commit(self, parents):
```
Comment (conradoverta) [inline client/verta/verta/_repository/repository.py]: Should we have a default value for parents?

Comment (convoliution) [reply client/verta/verta/_repository/repository.py]: Hmmm that makes sense. If the default value represents "no parents", that will replace the need for the `init()` function above (which is meant to create a `Commit` with no parents anyway).

File: client/verta/verta/_repository/commit.py [hunk 2]
```diff
@@ -29,13 +29,18 @@ def __init__(self, conn, repo_id, parent_ids=None, id_=None):
             self._update_blobs_and_parent_ids(parent_id)
 
     def __repr__(self):
-        header = "Commit containing:"
+        if self.id is None:
+            header = "unsaved Commit containing:"
```
Comment (conradoverta) [inline client/verta/verta/_repository/commit.py]: Random capitalization?

Comment (convoliution) [reply client/verta/verta/_repository/commit.py]: I've been consistently capitalizing things I consider to be Verta objects like "Project", "Experiment", "Experiment Run", "Repository" and "Commit". Would you not want this?

Comment (conradoverta) [reply client/verta/verta/_repository/commit.py]: Looks weird to me, but sure.

**Pull request merged**